ohttp/build.rs

```diff
@@ -275,7 +323,9 @@ mod nss {
             nsslibdir.to_str().unwrap()
         );
         if is_debug() {
-            static_link();
+            let use_static_softoken = false;
+            let use_static_nspr = false;
```

Comment:
These could both be set to true for the existing nss feature so that no LD_LIBRARY_PATH/dlopen stuff is needed.

Reply:
That's probably a good idea, yes.